handbook/engineering/deployments/index.md

```diff
@@ -5,20 +5,27 @@ We maintain multiple [instances](instances.md) of Sourcegraph:
 - [Instances](./instances.md): information about our different instances
   - [sourcegraph.com](instances.md#sourcegraph-com) is our production deployment for open source code.
   - [sourcegraph.sgdev.org](instances.md#sourcegraph-sgdev-org) is our private deployment of Sourcegraph that contains some of our private code.
-  - [k8s.sgdev.org](instances.md#k8s-sgdev-org) is a dogfood deployment that replicates the scale of our largest customers. Note that this also contains all of our private code.
-- [Managed instances](../distribution/managed/index.md) are deployments of Sourcegraph we manage for customers.
-  - [demo.sourcegraph.com](instances.md#demo-sourcegraph-copm) is a managed instance used for CE demos.
+  - [k8s.sgdev.org](instances.md#k8s-sgdev-org) is a dogfood deployment that replicates the scale of our largest customers. Note that this also contains all of our private code. - [Managed instances](../distribution/managed/index.md) are deployments of Sourcegraph we manage for customers.
+  - [demo.sourcegraph.com](instances.md#demo-sourcegraph-com) is a managed instance used for CE demos.
   - [devmanaged.sourcegraph.com](instances.md#devmanaged-sourcegraph-com) is a managed instance used for managed instances development.
 
 Learn more about how these work in:
 
-- [Deployment basics](#deployment-basics)
-- [Kubernetes](./kubernetes.md): setting up access, tips, and more
-- [Security](./security.md): TLS configuration,
-- [Testing](./testing.md): deploying test instances of Sourcegraph
-- [Playbooks](./playbooks.md): how-to guides for common tasks
-- [deploy-sourcegraph](#deploy-sourcegraph): base configuration for Kubernetes Sourcegraph deployments
-- [PostgresSQL](./postgresql.md): Tips for working with Postgres and our deployments
+- [Deployments](#deployments)
+  - [Deployment basics](#deployment-basics)
+    - [Kubernetes](./kubernetes.md): setting up access, tips, and more
+    - [Security](./security.md): TLS configuration,
```

Comment:
I think we should omit the top-level header here ("Deployments"), but I'm guessing this is VSCode trying to do its auto-TOC thing 😓

Reply:
This is the auto TOC thing 😬 I can drop this top-level header